### ISSUE TITLE
It fails on Rails 5 with LoadError: cannot load such file -- rails/test_unit/sub_test_task

### ISSUE THREAD

**Issue by rusanu** [rails/rails-perftest#33]
```
LoadError: cannot load such file -- rails/test_unit/sub_test_task
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:302:in `require'
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:302:in `block in require'
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:268:in `load_dependency'
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:302:in `require'
.../.rvm/gems/ruby-2.2.4/gems/rails-perftest-0.0.6/lib/rails/perftest/railties/testing.tasks:1:in `<top (required)>'
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:296:in `load'
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:296:in `block in load'
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:268:in `load_dependency'
.../.rvm/gems/ruby-2.2.4/gems/activesupport-5.0.0.beta2/lib/active_support/dependencies.rb:296:in `load'
.../.rvm/gems/ruby-2.2.4/gems/rails-perftest-0.0.6/lib/rails/perftest/railtie.rb:8:in `block in <class:Railtie>'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/railtie.rb:237:in `instance_exec'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/railtie.rb:237:in `block in run_tasks_blocks'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/railtie.rb:245:in `each'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/railtie.rb:245:in `each_registered_block'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/railtie.rb:237:in `run_tasks_blocks'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/application.rb:440:in `block in run_tasks_blocks'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/engine/railties.rb:13:in `each'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/engine/railties.rb:13:in `each'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/application.rb:440:in `run_tasks_blocks'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/engine.rb:457:in `load_tasks'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/railtie.rb:194:in `public_send'
.../.rvm/gems/ruby-2.2.4/gems/railties-5.0.0.beta2/lib/rails/railtie.rb:194:in `method_missing'

```

This is similar to https://github.com/drapergem/draper/issues/681 . 
Rails 5 no longer has `rails/test_unit/sub_test_task`

**Comment by senny** [comment]
yes, that's true. This gem currently does not work with Rails 5.

**Comment by senny** [comment]
@rusanu can you check to see if running the `master` version works for you?

**Comment by rusanu** [comment]
Thanks Yves, `master` works.

**Comment by sreuterle** [comment]
It would be great if you release a new gem version so that this gem works with latest rails version too.

**Comment by senny** [comment]
@sreuterle Thanks for the reminder. I have released `0.0.7`.
